packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -206,12 +213,19 @@ export class BacktraceDatabase implements BacktraceModule {
             const records = [...this._databaseRecordContext.getBucket(bucketIndex)];
 
             for (const record of records) {
+                if (!this.enabled) {
+                    return;
+                }
                 if (record.locked) {
                     continue;
                 }
                 try {
                     record.locked = true;
-                    const result = await this._requestHandler.send(record.data, record.attachments);
+                    const result = await this._requestHandler.send(
```

Comment:
I think we should also allow passing `AbortSignal` to the `send` method.

Reply:
good idea. thx